EIPS/eip-6963.md

```diff
@@ -39,27 +39,72 @@ Wallet: A user agent that manages keys and facilitates transactions with Ethereu
 
 Decentralized Application (DApp): A web page that relies upon one or many Web3 platform APIs which are exposed to the web page via the Wallet.
 
-Provider Discovery Library: A library or piece of software that assists a DApp to interact with the Wallet. 
+Provider Discovery Library: A library or piece of software that assists a DApp to interact with the Wallet (e.g. wagmi, Web3Modal, RainbowKit, Web3-Onboard, ConnectKit, etc).
 
 ### Provider Info
 
-Each wallet provider will be announced with the following interface `EIP6963ProviderInfo`. The values in the `EIP6963ProviderInfo` MUST be used as follows:
+Each Wallet Provider will be announced with the following interface `EIP6963ProviderInfo`. The values in the `EIP6963ProviderInfo` MUST be used as follows:
 
-- **`uuid`** - a globally unique of the wallet provider that MUST be (UUIDv4[^rfc4122] compliant) to uniquely distinguish different [EIP-1193](./eip-1193.md) provider sessions. The cryptographic uniqueness provided by UUIDv4[^rfc4122] guarantees that two independent `EIP6963ProviderInfo` objects can be separately identified.
```

Comment:
Just moved this line down to "Universally Unique Identifiers (UUIDs)" underneath (to keep consistent with the "Wallet IDs", "Wallet Names" and "Images/Icons" format)